cypress/commons/actions/generic/AppBar.js

```diff
@@ -26,7 +26,7 @@ function getHelpMenu() {
   return cy.get(GENERIC_SELECTORS.helpMenu.menu);
 }
 function getDocumentationLink() {
-  const docLink = `a[href="${FILE_NAME.DOC}"]`;
+  const docLink = `a[href="${DOCUMENTATION_URL}"]`;
   return cy.get(docLink);
```

Comment:
Can we use a different selector than the documentation URL ? (e.g. a data-cy)
Otherwise, our tests will be broken every time the URL changes